platform-api/src/dependencies.py

```
@@ -0,0 +1,222 @@
+        The D1 database proxy object. Use it with:
+        ``await db.prepare("SELECT * FROM users WHERE id = ?").bind(42).all()``.
+
+    Raises:
+        HTTPException(503): If the D1 binding is not configured in
+            ``wrangler.jsonc``.
+    """
+    if settings.db is None:
+        raise HTTPException(status_code=503, detail="D1 database not configured")
+    return settings.db
+
+async def get_sanity(settings: WorkerSettings = Depends(get_settings)) -> SanityClient:
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)